### PR TITLE
WSL support improvements

### PR DESCRIPTION
**Pull request by eugene-babichenko**
- Implement `__done_get_focused_window_id` on WSL as a PowerShell
  script.
- Replace `command "$powershell_exe"` with `eval` because a variable
  cannot be used as a command path.
- Use the `-Silent` option of BurntToast when `__done_notify_sound` is
  set to zero.

### CODE REVIEW

**Comment by franciscolourenco** [comment]
@eugene-babichenko thank you for the PR.
@ammgws do you have windows setup handy to test this? would be nice to have at least a 2nd person testing before publishing. thanks!

**Comment by ammgws** [comment]
@franciscolourenco 
I should be able to get access to a windows machine sometime this week.

File: conf.d/done.fish [hunk 1]
```diff
@@ -163,21 +191,18 @@ if test -z "$SSH_CLIENT" # not over ssh
...
-
-                    if test "$__done_notify_sound" -eq 1
-                        set soundopt "-Sound Default"
-                    end
-                    command "$powershell_exe" -command New-BurntToastNotification -Text \""$title"\",\""$message"\" $soundopt
-                end
+                __done_run_powershell_script '
+                    Import-Module -Name BurntToast 2>&1 | Out-Null
+                    if (Get-Module -Name BurntToast) {
+                        New-BurntToastNotification -Text "$title","$message" $soundopt
```
Comment (ammgws) [inline conf.d/done.fish:203]: Are these variables being expanded? AFAIK they won't be inside single quotes.

**Comment by ammgws** [comment]
OK so it looks like the window state detection is working fine (doesn't notify if window is focused), however notifications aren't working:
image

I'm guessing it's because the variables to BurntToast aren't being expanded due to the code block being in single quotes.

File: conf.d/done.fish (hunk 1 above)
Comment (eugene-babichenko) [reply conf.d/done.fish]: You are right. Fixed that.

**Comment by ammgws** [comment]
Can confirm it works when fixed to use double quotes.

**Comment by franciscolourenco** [comment]
Thank you for the awesome contribution!

**Pull request merged**